config/localization/en/app.json

```diff
@@ -1394,6 +1401,8 @@
       "POSTAL_CODE": "Postal Code",
       "PRINT_NAME": "Print Name",
       "REWARDS_BY_EPOCH": "Rewards by Epoch",
+      "REWARDS_LAST_7_DAYS": "Rewards last 7 days",
```

Comment:
don't think we need this anymore -- should be This week or This week rewards

Reply:
addressed in bb53067d74d4b1b48541e5e26847c81341be3130